packages/expo-clipboard/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "expo-clipboard",
+  "version": "1.0.0",
+  "description": "ExpoClipboard standalone module",
+  "main": "build/Clipboard.js",
+  "types": "build/Clipboard.d.ts",
+  "scripts": {
+    "build": "expo-module build",
+    "clean": "expo-module clean",
+    "lint": "expo-module lint",
+    "test": "expo-module test",
+    "prepare": "expo-module prepare",
+    "prepublishOnly": "expo-module prepublishOnly",
+    "expo-module": "expo-module"
+  },
+  "keywords": [
+    "react-native",
+    "expo",
+    "expo-clipboard"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/expo/expo.git",
+    "directory": "packages/expo-clipboard"
+  },
+  "bugs": {
+    "url": "https://github.com/expo/expo/issues"
+  },
+  "author": "650 Industries, Inc.",
+  "license": "MIT",
+  "homepage": "https://docs.expo.io/versions/latest/sdk/expo-clipboard",
+  "unimodulePeerDependencies": {
+    "@unimodules/core": "*"
+  },
+  "peerDependencies": {
+    "react": "*",
+    "react-native": "*"
+  },
+  "dependencies": {
+    "expo-module-scripts": "^1.2.0"
```

Comment:
I don't think this should be part of the `dependencies`, right? This should be a `devDependency`, we use it only in development.